### DISCUSSION TITLE
월드 오브젝트에 노멀맵을 적용할 수 있을까요?

### DISCUSSION THREAD

**Discussion by tooon2602** [naverz/zepeto-studio-kor#1260] [5. Q&A]
아이템은 노멀맵으로 재질감을 입힐 수 있는데 월드 오브젝트에 노멀맵을 적용할 수 있을까요? 월드 최적화를 위해 노멀맵 사용이 가능한지 궁금합니다.

**Comment by jefflim69** [comment]
제 경험으로 말씀 드리면, Albedo , Metalic , Normal 모두 디바이스에서 정상적으로 나옵니다 .

**Reply by tooon2602** [reply]
답변 감사합니다. 혹시 노멀맵 사용이 월드 최적화에 도움이 되었을까요? 다른월드 보니까 메쉬로 만들고 이미지 텍스처만 사용한 것 같아서 고민되네요.

**Reply by jefflim69** [reply]
최적화의 요인은  Normal Map 이외에도 , Polygon 갯수, Texture 갯수, 크기 등 다양한 요인이 있으니...단순히 Normal Map 때문에
고민하실 일은 아닌 것 같네요 . Device 에서 Test 했을 때 구동이 원활하다면 문제가 없다고 보시면 될 것 같습니다 .

**Comment by YSHFM** [comment]
https://www.mettit.com/challenge/applcn-view/CPN20220816160051228

제가 타 커뮤니티에서 작성한 글입니다만 보시다시피 제페토에서 정상적으로 나옵니다.
